charts/rancher-k3s-upgrader/0.3.2/templates/deployment.yaml

```diff
@@ -14,6 +14,13 @@ spec:
     spec:
       affinity:
         nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
+                  - key: "kubernetes.io/os"
+                    operator: In
```

Comment:
what about using the `NotIn` operator instead? That's what we use in other rancher charts

```
                  operator: NotIn
                  values:
                    - windows
```